protocol/communities_messenger_test.go

```diff
@@ -3684,3 +3684,170 @@ func (s *MessengerCommunitiesSuite) TestRetrieveBigCommunity() {
 	}, "updated description not received")
 	s.Require().NoError(err)
 }
+
+func (s *MessengerCommunitiesSuite) TestRequestAndCancelCommunityAdminOffline() {
```

Comment:
I checked myself and the test seems to be flaky (possible culprits in other comments). It is crucial to have in mind that _messages in status-go tests may come out of order_.

In order to test if test is flaky you could use `-count=20`:
`go test -timeout 60s -tags gowaku_skip_migrations -count=20 -run ^TestMessengerCommunitiesSuite$ -testify.m ^\(TestRequestAndCancelCommunityAdminOffline\)$ github.com/status-im/status-go/protocol`